SDKTest/NeuroIDClassTests.swift

```diff
@@ -34,7 +35,22 @@ class NeuroIDClassTests: XCTestCase {
         // Clear out the DataStore Events after each test
         clearOutDataStore()
     }
-
+    
+   
+
+    func test_getAdvDeviceLatency() {
+        let mockService = MockDeviceSignalService()
+        NeuroID.deviceSignalService = mockService
+        _ = NeuroID.configure(clientKey: "key_test_0OMmplsawAp2CQfWrytWA3wL")
+        UserDefaults.standard.removeObject(forKey: Constants.storageAdvancedDeviceKey.rawValue)
+        let randomTimeInMilliseconds = Double(Int.random(in: 0..<3000))
+        mockService.mockResult = .success(("empty mock result. Can be filled with anything", randomTimeInMilliseconds))
+        _ = NeuroID.start(true)
+        let allEvents = DataStore.getAllEvents()
+        let validEvent = allEvents.filter { $0.type == "ADVANCED_DEVICE_REQUEST" }
+        XCTAssertTrue(validEvent.count == 1)
```

Comment:
NIT: Can we add spacing to separate the setup vs. test vs. assertion for readability?